roles/internal/fcrepo/defaults/main.yml

```diff
@@ -3,3 +3,5 @@ fcrepo_user: tomcat8
 fcrepo_data_dir: /var/lib/tomcat8/fcrepo4-data
 fcrepo_war_path: "{{ tomcat8_home }}/webapps/fcrepo.war"
 fcrepo_home_dir: /opt/fcrepo
+fcrepo_activemq_template: activemq.xml.j2
+fcrepo_tomcat_config_dir: "{{ tomcat8_home }}/webapps/fcrepo/WEB-INF/classes/config/"
```

Comment:
It's generally not best practice to put config like this inside the filesystem space of the webapp itself... maybe someplace inside `/usr/local` or `/etc` or the like?

Reply:
I agree with you 100%. Fedora itself puts a `activemq.xml` file at this path, that I am trying to override here. Is there some way to pass a path for this config to fedora so it reads the alternative configuration? I'm not sure how I would do that.

Reply:
Jeez, I can't believe Fedora does that. No, I can totally believe that. If that's the way that Fedora does it, I guess we're probably better off leaving it alone.

Reply:
I think we reverted my work on Fedora to make a single configuration file that you could override with a JAVA_OPT. I'll look back and see.

Reply:
Ok, I think we can put the `activemq.xml` anywhere and set use this https://github.com/fcrepo4/fcrepo4/blob/fcrepo-4.7.4/fcrepo-webapp/src/main/resources/spring/jms.xml#L18-L19

Reply:
So I'm not sure how we do this in Ansible, but we need to add
`-Dfcrepo.activemq.configuration=file:/blah/blah/activemq.xml`
to
https://github.com/Islandora-Devops/claw-playbook/blob/master/inventory/vagrant/group_vars/tomcat.yml#L9-L22

Reply:
Cool. Thanks @whikloj I'll update the playbook.